youtube_dl/utils.py

```diff
@@ -3901,10 +3954,10 @@ def urlencode_postdata(*args, **kargs):
 def update_url_query(url, query):
     if not query:
         return url
-    parsed_url = compat_urlparse.urlparse(url)
-    qs = compat_parse_qs(parsed_url.query)
+    parsed_referrer_source = compat_urlparse.urlparse(url)
+    qs = compat_parse_qs(parsed_referrer_source.query)
     qs.update(query)
-    return compat_urlparse.urlunparse(parsed_url._replace(
+    return compat_urlparse.urlunparse(parsed_referrer_source._replace(
```

Comment:
don't make unrelated changes.

Reply:
This must have changed when updating `get_referrer_url` to match the specification.